<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenuScene</class>
 <widget class="QWidget" name="MainMenuScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>351</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="inquiryButton">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Account Inquiry</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="registerButton">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Register Account</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="depositButton">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Deposit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="withdrawButton">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Withdraw</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="goBackButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>500</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Go Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Main Menu</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
